### ISSUE TITLE
Allow splitting the editor vertically instead of horizontally

### ISSUE THREAD

**Issue by ldorigo** [MacDownApp/macdown#937]
This would really help when working side-by-side with another app on a smaller monitor.

By "splitting", I mean having the editor on top and the preview on bottom (or the opposite).

Great work with your app, I love it!

**Comment by uranusjr** [comment]
I’m not keen on this feature myself, but would be happy to accept contribution if anyone is interested in this. Likely this involves refactoring the current split pane logic quite a bit.

**Comment by franklinyu** [comment]
> Likely this involves refactoring the current split pane logic quite a bit.

+1

**Comment by SimplGy** [comment]
Is it the synchronized scrolling, in particular, that would make this difficult?

This feature doesn't make a ton of sense unless you have a vertically oriented monitor -- **then** it becomes really useful. :)

**Comment by SimplGy** [comment]
Oh, that seems like the easy part! What buttons would become dead? MacDown is pretty button-free. Maybe there's a toolbar I don't use?

Are we doing layouts with cassowary style constraints or struts/springs so layouts are flexible across devices and screens?

**Comment by franklinyu** [comment]
@SimplGy Yes, there is a toolbar, but it has not yet been released. It was implemented in #901 and currently in master branch. See #282 for history and more information. Here is a screenshot from #282:

![MacDown with toolbar](https://user-images.githubusercontent.com/6615076/33340786-5f262ca4-d47d-11e7-8251-b4bae0590111.png)

Note the top-right corner.

**Comment by SimplGy** [comment]
Ah! I see.

Those are beautiful icons, thanks for sharing.

Looks like the work effort is to come up with a great-looking, intuitive way to support this same functionality and also support splitting?

**Comment by Turing-Chu** [comment]
if macdown support split window，one window using for  edit markdown statement, another window using for showing real effect.

![image](https://user-images.githubusercontent.com/13805470/58425000-4b352000-80cb-11e9-8485-9c38aefde766.png)

**Comment by FranklinYu** [comment]
@Turing-Chu That’s another feature request, not related to this.
